dir.traversal.targets

```diff
@@ -86,6 +86,32 @@
              ContinueOnError="ErrorAndContinue" />
   </Target>
 
+  <Target Name="ProducesPackageId"
+          Returns="@(PackageIds)" 
+          DependsOnTargets="FilterProjects">
+        <MSBuild Targets="ProducesPackageId"
+             Projects="@(Project)"
+             BuildInParallel="true"
+             ContinueOnError="ErrorAndContinue">
+          <Output TaskParameter="TargetOutputs"
+                  ItemName="PackageIds" />             
+        </MSBuild>
+    <ItemGroup>
+      <PackageIds Include="@(PackageIds)" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="ProducesTarget" DependsOnTargets="FilterProjects">
```

Comment:
I had intended to name the entry target for Produces to be "Produces", but there's a bug in the run tool such that you can't name an MSBuild target and a run command the same thing.  @maririos